### DISCUSSION TITLE
Unable to install react-native-reanimated on Expo

### DISCUSSION THREAD

**Discussion by Th0rgal** [software-mansion/react-native-reanimated#5523] [Q&A]
Hello,

I have an existing Expo project and I wanted to install `[react-native-reanimated](https://github.com/software-mansion/react-native-reanimated)` because it is required by this library I would like to use: https://gorhom.github.io/react-native-bottom-sheet/

It's my first time with react-native and I enjoyed the experience so far, but after following the installation instructions and trying to run my app on ios, I got:
```
iOS Bundling complete 4431ms
 ERROR  Error: Failed to create a worklet. Did you forget to add Reanimated Babel plugin in babel.config.js? See installation docs at https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation#babel-plugin., js engine: hermes
 ERROR  Invariant Violation: "main" has not been registered. This can happen if:
* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
* A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called., js engine: hermes
```

I couldn't run `cd ios && pod install` as explained on the docs as I don't have an ios folder, I guess it is not required when using expo.

Here is my `babel.config.js`:
```js
module.exports = function(api) {
  api.cache(true);
  return {
    presets: ['babel-preset-expo'],
    plugins: ['react-native-reanimated/plugin'],
  };
};
```

### Repository for Reproduction
The issue can be reproduced using my project available at: [https://github.com/Th0rgal/dietgpt](https://github.com/Th0rgal/dietgpt)

### Attempted Solutions
- Verified dependencies.
- Cleared caches and restarted.
- Reinstalled dependencies.

**Comment by MatiPl01** [comment]
Hi! Your babel config looks correct. May I ask you which version of `react-native-reanimated` you installed? Libraries often aren't up-to-date on expo and you may have to use the older version than the latest one. Currently the latest supported version for expo is `3.3.0`.

I cannot access the reproduction project that you linked in the description so it's hard to tell what's wrong with your setup.

**Comment by MatiPl01** [comment]
This discussion seems to be stale.

I am closing it for now. If you have more questions, don't hesitate to re-open the discussion.
